applications/app/controllers/SignupPageController.scala

```diff
@@ -24,23 +27,64 @@ class SignupPageController(
     with GuLogging {
 
   val defaultCacheDuration: Duration = 15.minutes
+  val remoteRenderer = DotcomRenderingService()
 
-  def renderNewslettersPage(): Action[AnyContent] =
+  private def getShouldUseRemoteRender()(implicit
+      request: RequestHeader,
+  ): Boolean = {
+    request.getQueryString("dcr").contains("true")
+  }
```

Comment:
As I understand we shouldn't need this function, you should be able to use the `request.forceDCR` property as this is an [implicit added to RequestHeader](https://github.com/guardian/frontend/blob/main/common/app/implicits/Requests.scala#L112).

Reply:
Thanks - I wasn't familiar with implicits, but I'll push a change to make use of it.